terraform/modules/monitoring/variables.tf

```diff
@@ -102,8 +102,8 @@ variable "log_level" {
 }
 
 variable "pmap_subscription_ids" {
-  type        = list(string)
-  default     = []
+  type        = map(string)
```

Comment:
I think a list will be more user friendly.

Reply:
Terraform will complain and throw out errors if it’s a list and says it prefers a map. I have tried using a list, it didn’t work, so I changed to this.

Reply:
If map isn’t wanted, I can try if set works.

Reply:
I ran terraform apply and it worked without any warning.

Reply:
Discussed offline, the original module should work as I changed the list to set in the monitoring module.

Reply:
Updated.

Reply:
Discussed offline, reproduced the real error, and decided to use map and some comments to describe why we used map here.